### PR TITLE
adds 2 slim shortcodes

### PR DESCRIPTION
**Pull request by binarygary**
This adds 2 shortcodes for a slim version of the genrenator and the storynator.

### CODE REVIEW

**Review by jazzsequence — COMMENTED** [review]
These are okay but why use a shortcode to just return the story/genre when you could use the API output? Just wondering how this would be used in practice.

**Comment by binarygary** [comment]
@jazzsequence In the context of the slack api oauth signup process I need to provide a success and failure page. I thought it would be fun to include a genre dynamically on either of them. Shortcode seemed like the fasted way to do so.

see: http://binaryjazz.us/success/

**Comment by binarygary** [comment]
@jazzsequence I also am slipping in a generated genre counter. 
It could be fun to use on the genrenator app page once the number is large.

**Comment by binarygary** [comment]
Last update...couldn't work on this without adding more instruments.

File: inc/genrenator.php [hunk 1]
```diff
@@ -69,6 +69,8 @@ function generate_genre() {
 	$patterns = patterns();
 	$index    = array_rand( $patterns );
 
+	update_option( 'generate_genre', get_option( 'generate_genre', 0 ) + 1, false );
+
```
Comment (jazzsequence) [inline inc/genrenator.php]: I feel like if we’re adding an option here, we should have an action hook along with it.

**Review by jazzsequence — DISMISSED** [review]
This all looks good. We would want to make sure the counter increments properly when generating multiple genres, but I think what you have will work. I think that we should start adding hooks and the counter seems like a good place to put a do_action.

**Comment by jazzsequence** [comment]
Should we add a shortcode for the counter, too? So we can throw that on the success page with something like `975 genres created so far`?

**Comment by jazzsequence** [comment]
@binarygary mind sanity checking my updates?

**Pull request merged**